project3/connect4_frontend/src/main.rs

```diff
@@ -1,9 +1,24 @@
 use yew::prelude::*;
-
+use std::io;
 
 mod board;
 use board::*;
 
+mod fsm;
+use fsm::*;
+
+mod boardgame;
+use boardgame::*;
+
+mod chip;
+use chip::*;
+
+mod ai;
+use ai::*;
```

Comment:
is there an ai.rs file that I'm missing?

Reply:
Just added it. I indeed forgot to commit the ai.rs file